packages/material-ui/src/LinearProgress/LinearProgress.js

```diff
@@ -43,6 +43,7 @@ export const styles = theme => ({
     width: '100%',
     animation: 'buffer 3s infinite linear',
     animationName: '$buffer',
+    willChange: 'opacity, background-position',
```

Comment:
So how do we determine when to use it? "Opinionated" is to vague IMO. I thought we simply remove every usage.

Reply:
What does "continuously running" entail? `will-change` got removed in `Slider` too which would fit that description.

Reply:
Maybe we should remove it everywhere then?

Reply:
I think so too after reading that documentation page. It's a "last resort" and should not be applied prematurely. I would hope that a single component hits 60fps anyway which is why application authors need to benchmark their ui not library authors a single component WRT to `will-change`.

Reply:
Slider only has transformations when active, The indeterminate progress components have animations that continuously loop

Reply:
Running animations are promoted to a layer automatically and thus does not need will-change. Will-change is, as it's name implies for things that are not currently changing but is about to so that the animation doesn't have to start with a costly layer promotion.